lib/stencil-bundle.js

```diff
@@ -82,6 +82,9 @@ Bundle.prototype.initBundle = function () {
 
 Bundle.prototype.getCssAssembleTask = function (compiler) {
     var self = this;
+    var assembleOptions = {
```

Comment:
If this is a constant why do you need to add it as an option and add a 5th parameter?